dts/bindings/sram/mmio-sram.yaml

```diff
@@ -10,3 +10,7 @@ include: base.yaml
 properties:
     reg:
       required: true
+
+    pmc-communication:
```

Comment:
What is this property and why are you adding it to the base `mmio-sram` binding?